### ISSUE TITLE
System tokens

### ISSUE THREAD

**Issue by alexeyshmelev** [kuleshov-group/caduceus#73]
Hi, I’m using your tokenizer from Huggingface, and it only uses tokens for nucleotides, as well as the [PAD] and [SEP] tokens, but it never uses the [CLS] or [BOS] tokens. Am I doing everything correctly?

**Comment by yair-schiff** [comment]
Can I clarify what you mean by:
> it never uses the [CLS] or [BOS] tokens. Am I doing everything correctly?

In general though, I don't think we used the special tokens you have listed, since we trained on sequences that fit the entire context length and performed Masked Language Modeling to fill in [MASK] tokens

**Comment by alexeyshmelev** [comment]
@yair-schiff, when fine-tuning your model for downstream tasks, I usually paste the DNA sequence into your tokenizer, and it returns tokens in the following order: padding tokens → nucleotide tokens → sep token. Is this correct? Should I manually insert `[CLS]` or `[BOS]` tokens, or is this unnecessary for your model? Of course, I can add them myself, but do I actually need to include them for tasks like token classification? Maybe you used `[CLS]` or `[BOS]` tokens during pretraining phase and it can affect the performance of the model?
